db/external/binaries-list

```diff
@@ -1,2 +1,18 @@
-F787C9B484CD7526F866C21D8925C4DACE467F8A derby-10.2.2.0.jar
-97771BE04E7452FC197EB875D2591A7E91F274D0 derby-10.2.2.0.zip
```

Comment:
The zip is unused, right? (I did a quick search, and didn't find a place where it would be used.)

Reply:
I did not find any usage and as such I removed that reference.